### PR TITLE
cyclic, cmd: qoute cyclic table name, fatal if loopback and fix a data race

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

- Correct a few logs
- Fix data race on log
- Fail fast if loopback detected
- Quote mark table name in update SQL.

### Check List 

Tests 

 - Integration test

Related changes

 - Need to cherry-pick to the release branch

### Release note

- Fix the issue log may cause data race.

### CODE REVIEW

**Comment by overvenus** [comment]
Base on prometheus spec, label name must match the regex `[a-zA-Z_:][a-zA-Z0-9_:]*`. We need to sanitize table names.

File: tests/cyclic_loopback/run.sh [hunk 1]
```diff
@@ -1,66 +0,0 @@
-#!/bin/bash
```
Comment (amyangfei) [inline tests/cyclic_loopback/run.sh:1]: why is this test case removed?

**Pull request merged**